src/config.json

```diff
@@ -17,5 +17,6 @@
     "logLevel": "debug",
     "imageProxyURL": "",
     "imageAPIKey": "",
-    "customProtocols": ["tezori", "tezos", "galleon"]
+    "wertUrl": "https://widget.wert.io/01EMHNZQFMYEPJACS7B6FFH09Y/widget/login?click_id=12345&commodity=XTZ&address=",
```

Comment:
Should we not leave this blank and override it internally? @anonymoussprocket

Reply:
that's fair, we do that to the rest of the URLs